sklearn/tests/test_multiclass.py

```diff
@@ -605,3 +605,13 @@ def test_ecoc_gridsearch():
     cv.fit(iris.data, iris.target)
     best_C = cv.best_estimator_.estimators_[0].C
     assert_true(best_C in Cs)
+
+def test_pairwise_attribute():
```

Comment:
Do you mean add a test similar to the one here? 

https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tests/test_cross_validation.py#L714